plugins/generic.js

```diff
@@ -79,6 +85,11 @@ class Generic {
     // TODO: detect cycle to avoid infinite navigation for frameworks
     // that support loopable presentations like impress.js and flowtime.js
     await pause(1000);
+
+    if (!this.isNextSlideDetected && (typeof this.secondaryKey !== 'undefined')) {
+      await this.page.keyboard.press(this.secondaryKey);
```

Comment:
Intuitively, I’d expect the « secondary » key, e.g., the down arrow key, to be used first, and if no changes are detected, then fall through using the « primary » key, i.e., the right arrow key.

Reply:
Sure, we can reverse the order.